### PR TITLE
Add kubectl plugin with basic command

### PR DESCRIPTION
**Pull request by chiayi**
## Checks

- [ ] I've made sure the tests are passing.
- Testing Strategy
   - [ ] Unit tests
   - [x] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: kubectl-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Kuberay Kubectl Plugin
+
+Kubectl plugin/extension for Kuberay CLI that provides the ability to manage ray resources.
+
+    The kubectl plugin is an optional plugin that has a dependency on the KubeAPI server.
```
Comment (andrewsykim) [inline kubectl-plugin/README.md]: It doesn't depend on KubeAPI Server

File: kubectl-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    Aliases:
+    get, list
+
+    Flags:
+    -A, --all-namespaces                 If present, list the requested clusters across all namespaces. Namespace in current context is ignored even if specified with --namespace.
+        --as string                      Username to impersonate for the operation. User could be a regular user or a service account in a namespace.
+        --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
+        --as-uid string                  UID to impersonate for the operation.
+        --cache-dir string               Default cache directory (default "/Users/aaronliang/.kube/cache")
```
Comment (chiayi) [inline kubectl-plugin/README.md]: Change the dir @chiayi

File: kubectl-plugin/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Installation
+
+<!-- 1. Check [release page](https://github.com/ray-project/kuberay/releases) and download the necessary binaries. -->
+1. Run `go build cmd/kubectl-ray.go`
+2. Move the binary, which will be named `kubectl-ray` to your `PATH`
+
+## Prerequisites
+
+1. Make sure there is a Kubernetes cluster running with a Kuberay cluster.
```
Comment (andrewsykim) [inline kubectl-plugin/README.md]: Make sure there is a Kubernetes cluster running with KubeRay installed.

File: kubectl-plugin/pkg/cmd/ray.go [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+const (
+	CLIName = "ray"
+)
+
+func NewRayCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "ray",
+		Short:        "ray kubectl plugin",
+		Long:         `Manage ray resources and eliminate the need for user to handle long yamls.`,
```
Comment (andrewsykim) [inline kubectl-plugin/pkg/cmd/ray.go]: I think just "Manage RayCluster resources" is fine

**Pull request merged**